cmd/blockchaincmd/deploy.go

```diff
@@ -175,13 +175,8 @@ so you can take your locally tested Blockchain and deploy it on Fuji or Mainnet.
 
 	icmGroup := flags.RegisterFlagGroup(cmd, "ICM Flags", "show-icm-flags", false, func(set *pflag.FlagSet) {
 		set.BoolVar(&icmSpec.SkipICMDeploy, "skip-icm-deploy", false, "Skip automatic ICM deploy")
-		set.BoolVar(&icmSpec.SkipICMDeploy, "skip-local-teleporter", false, "skip automatic ICM deploy on local networks [to be deprecated]")
-		set.BoolVar(&icmSpec.SkipICMDeploy, "skip-teleporter-deploy", false, "skip automatic ICM deploy")
-		set.BoolVar(&icmSpec.SkipICMDeploy, "noicm", false, "skip automatic ICM deploy")
```

Comment:
isn't --noicm better that any other option? or --icm

Reply:
noicm is less clear than skip-icm-deploy